api/models/post.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const postSchema = Schema({
+  author: [
+    {
+      ref: "User",
+      type: Schema.Types.ObjectId,
+      required: true,
+    },
+  ],
+
+  date: {
+    type: String,
+    required: true,
+  },
+
+  title: {
+    type: String,
+    required: true,
+  },
+
+  content: {
+    type: String,
+    required: true,
+  },
+
+  likes: {
+    type: Number,
```

Comment:
Could be ref to users that liked this post

Reply:
Changed